keyboards/gergo/keymaps/default/readme.md

```diff
@@ -0,0 +1,8 @@
+# [Gergo! By g Heavy Industries](http://gboards.ca)
+
+![Gergo image](https://4.bp.blogspot.com/-889nMXxgSM0/XCNxwnO5kUI/AAAAAAAA6mI/tZbWgZVCBW0dyZOCGJDkjN06DVax7j8XwCLcBGAs/s1600/48422820_967732713413298_485744639215665152_n.jpg)
+
+This is the default keymap for Gergo, it's based heavily off of the naps62 ErgoDox layout and is aimed at a programmer friendly keymap.
+
+## Settings
+To edit various settings, enable the 1u trackball and whatnot please modify /keyboards/gergo/rules.mk
```

Comment:
This should actually be done at the keymap level (`keyboards/gergo/keymaps/<keymapname>/rules.mk`). Changing it at the keyboard level changes it for everyone who uses the board and merges their keymap into qmk/qmk_firmware.

```suggestion
To edit various settings, enable the 1u trackball and whatnot please modify `keyboards/gergo/keymaps/<keymapname>/rules.mk`.
```

Along those lines, you should probably duplicate the rules needed for the necessary options to `keyboards/gergo/keymaps/default/rules.mk`, that way someone who wants to build their keymap already has a sample file to modify to their liking.

Reply:
Merged suggested changes!